components/automate-ui/src/app/entities/projects/project.effects.ts

```diff
@@ -196,7 +202,19 @@ export class ProjectEffects {
           observableOf(new GetApplyRulesStatusFailure(error))))));
 
   @Effect()
-  getLatestApplyRulesStatus$ = observableInterval(1000 * POLLING_INTERVAL_IN_SECONDS)
-    .pipe(map(() => new GetApplyRulesStatus()));
+  getLatestApplyRulesStatus$ = observableInterval(1000 * POLLING_INTERVAL_IN_SECONDS).pipe(
+    withLatestFrom(this.store.select(iamMajorVersion).pipe(filter(identity))),
+    withLatestFrom(this.store.select(iamMinorVersion).pipe(filter(identity))),
+    withLatestFrom(this.store.select(allPerms)),
+    switchMap(([[[_, major], minor], perms]) => {
+      const permObject = perms[ProjectConstants.APPLY_RULES_ENDPOINT];
+      const allowed = permObject && permObject.get;
+      if (allowed && major === 'v2' && minor === 'v1') {
+        return this.requests.getApplyRulesStatus().pipe(
+          map(() => new GetApplyRulesStatus()));
+      } else {
+        return observableOf(new NoopAction());
```

Comment:
❓ Can we not just _do nothing_? I'd think somehow we should be able to pipe `(allowed && major === 'v2' && minor === 'v1')` and filter based on that....?

Reply:
Great suggestion; see latest commit!